syftlib/src/main/java/org/openmined/syft/SignallingClient.kt

```diff
@@ -0,0 +1,141 @@
+package org.openmined.syft
+
+import android.util.Log
+import com.google.gson.JsonObject
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Flowable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.net.Socket
+import java.nio.charset.Charset
+import java.util.concurrent.Executors
+import java.util.concurrent.Future
+import java.util.concurrent.ScheduledFuture
+import java.util.concurrent.TimeUnit
+
+
+const val SOCKET_PING = "socket-ping"
+private const val TAG = "WebRTC Signalling Client"
+
+class SignallingClient(
```

Comment:
Could you add some tests to this class?

Reply:
Sure will do! I was thinking of doing them along with the demo app issue since both would be for testing